src/device.spec.ts

```diff
@@ -62,31 +61,5 @@ describe('device', function device() {
     });
 
     this.timeout(20000);
-
-    describe('deploy', () => {
-        it('works', async () => {
-            options.retainDeploymentArchive = true;
-            let response = await rokuDeploy.deploy(options as any);
-            assert.equal(response.message, 'Successful deploy');
-        });
-
-        it('Presents nice message for 401 unauthorized status code', async () => {
-            this.timeout(20000);
-            options.password = 'NOT_THE_PASSWORD';
-            await expectThrowsAsync(
-                rokuDeploy.deploy(options as any),
-                'Unauthorized. Please verify username and password for target Roku.'
-            );
-        });
-    });
-
-    describe('deployAndSignPackage', () => {
-        it('works', async () => {
-            await rokuDeploy.deleteInstalledChannel(options as any);
-            await rokuDeploy.rekeyDevice(options as any);
-            expectPathExists(
-                await rokuDeploy.deployAndSignPackage(options as any)
-            );
-        });
-    });
+    console.log(options); // So there are no errors about unused variable
```

Comment:
If we're not actually using options, let's delete it. Either way, delete this comment.

Reply:
done